submissions/acerpo/index.html

```diff
@@ -0,0 +1,66 @@
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style/style.css">
+
+  <title>HTML-CSS-Popup</title>
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <nav class="nav">
+                <div class="nav__item">
+                    <a href="#" class="headerLink">Mail</a>
+                </div>    
+                <div class="nav__item">
+                    <a href="#" class="headerLink">Image</a>
+                </div>   
+                <div class="nav__item popup">
+                    <input class="checkbox_input checkbox_popup" type="checkbox" id="checkbox_toggle">
+                    <label class="check headerLink" for="checkbox_toggle"><img class="nav__item_img" src="./img/popup-button.png" alt="popup_content"></label>
+                    <div class="popup_content">
+                        <div class="popup_content_body">
+                            <a class="popup_content_body_link" href="#"><img src="./img/calendar.png" alt="Calendar">Calendar</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/mail.png" alt="Mail">Mail</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/news.png" alt="News">News</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/maps.png" alt="Map">Map</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/files.png" alt="Files">Files</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/contacts.png" alt="Contacts">Contacts</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/tanslation.png" alt="Translate">Translate</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/videos.png" alt="Videos">Videos</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/search.png" alt="Search">Search</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/photos.png" alt="Photo">Photo</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/apps.png" alt="Apps">Apps</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/documents.png" alt="Document">Document</a>
+                            <div class="more">
+                                <input class="checkbox_input checkbox_more" type="checkbox" id="checkbox_toggle_more">
+                                <label class="more_label" for="checkbox_toggle_more">More</label>
+                                <div class="more_content">
+                                    <div class="more_content_body">
+                                        <a class="popup_content_body_link" href="#"><img src="./img/chat.png" alt="Chat">Chat</a>
```

Comment:
For icons better set an empty `alt` attribute.
Sorry for russian but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)